backend/apps/skills_matching/views/skill_tree_views.py

```
@@ -0,0 +1,349 @@
+    {"id": "open_source", "name": "Open Source Workflow", "count": 5},
+    {"id": "frontend", "name": "Frontend Track (React / TS)", "count": 3},
+    {"id": "backend", "name": "Backend Track (Django / Python)", "count": 2},
+    {"id": "devops", "name": "DevOps & CI/CD", "count": 1},
+]
+
+
+class SkillTreeViewSet(viewsets.ViewSet):
+    """
+    API ViewSet for Interactive Contributor Skill Tree & Dynamic Mastery Graph UI (#2323).
+    """
+    permission_classes = [AllowAny]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)